app/src/organisms/ChooseProtocolSlideout/index.tsx

```diff
@@ -141,10 +181,234 @@ export function ChooseProtocolSlideoutComponent(
+            </StyledText>
+            <Flex
+              gridGap={SPACING.spacing8}
+              justifyContent={JUSTIFY_FLEX_START}
+              width="max-content"
+            >
+              <ToggleButton
+                toggledOn={runtimeParam.value as boolean}
+                onClick={() => {
+                  const clone = runTimeParametersOverrides.map(
+                    (parameter, i) => {
+                      if (i === index) {
+                        return {
+                          ...parameter,
+                          value: !parameter.value,
+                        }
+                      }
+                      return parameter
+                    }
+                  )
+                  if (setRunTimeParametersOverrides != null) {
+                    setRunTimeParametersOverrides(clone)
+                  }
+                }}
+                height="0.813rem"
+                label={
+                  runtimeParam.value
+                    ? t('protocol_details:on')
+                    : t('protocol_details:off')
+                }
+                paddingTop={SPACING.spacing2} // manual alignment of SVG with value label
+              />
+              <StyledText as="p">
+                {runtimeParam.value
+                  ? t('protocol_details:on')
+                  : t('protocol_details:off')}
+              </StyledText>
+            </Flex>
+            <StyledText as="label" paddingTop={SPACING.spacing8}>
+              {runtimeParam.description}
+            </StyledText>
+          </Flex>
+        )
+      }
+    }) ?? null
+
+  const pageTwoBody = (
+    <Flex flexDirection={DIRECTION_COLUMN}>
+      <Flex justifyContent={JUSTIFY_END}>
+        <LinkComponent
+          textAlign={TYPOGRAPHY.textAlignRight}
+          css={
+            isRestoreDefaultsLinkEnabled ? ENABLED_LINK_CSS : DISABLED_LINK_CSS
+          }
+          onClick={() => {
+            const clone = runTimeParametersOverrides.map(parameter => ({
+              ...parameter,
+              value: parameter.default,
+            }))
+            if (setRunTimeParametersOverrides != null) {
+              setRunTimeParametersOverrides(clone)
+            }
+          }}
+          paddingBottom={SPACING.spacing10}
+          {...targetProps}
+        >
+          {t('protocol_details:restore_defaults')}
+        </LinkComponent>
+        {!isRestoreDefaultsLinkEnabled && (
+          <Tooltip tooltipProps={tooltipProps}>
+            {t('protocol_details:no_custom_values')}
+          </Tooltip>
+        )}
+      </Flex>
+      <Flex flexDirection={DIRECTION_COLUMN} gridGap={SPACING.spacing16}>
+        {runTimeParametersInputs}
+      </Flex>
+    </Flex>
+  )
+
+  const singlePageFooter = (
+    <PrimaryButton
+      onClick={handleProceed}
+      disabled={isCreatingRun || selectedProtocol == null}
+      width="100%"
+    >
+      {isCreatingRun ? (
+        <Icon name="ot-spinner" spin size={SIZE_1} />
```

Comment:
Sorry, I think we haven't shared the story of `SIZE_number`
SIZE consts haven't been updated for long time and we decided to use the actual number for `size`. 

```suggestion
        <Icon name="ot-spinner" spin size="1rem" />
```